src/libraries/System.Net.WebSockets.Client/tests/ConnectTest.Http2.cs

```diff
@@ -19,19 +19,29 @@ public class ConnectTest_Http2 : ClientWebSocketTestBase
     {
         public ConnectTest_Http2(ITestOutputHelper output) : base(output) { }
 
-        [Fact]
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
         [SkipOnPlatform(TestPlatforms.Browser, "Self-signed certificates are not supported on browser")]
-        public async Task ConnectAsync_VersionNotSupported_Throws()
+        public async Task ConnectAsync_VersionNotSupported_Throws(bool useHandler)
```

Comment:
Should we also test for HttpClient, not only HttpMessageInvoker?

Reply:
I'm planning to add it for all existing tests in scope of another issue https://github.com/dotnet/runtime/issues/72476